app/src/main/java/org/simple/clinic/storage/Migration_45_46.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.storage
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Suppress("ClassName")
+class Migration_45_46 : Migration(45, 46) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.execSQL("""
+      CREATE TABLE IF NOT EXISTS `HomescreenIllustration` (
```

Comment:
Use double quotes instead of back-ticks please. Back-ticks are a MySQL compatibility feature and not a SQL standard.

Reply:
done